src/ini.pest

```diff
@@ -1,17 +1,25 @@
-ini = { SOI ~ bom? ~ (pair | comment | NEWLINE)* ~ section* ~ EOI }
-bom = _{ "\u{feff}" }
-WHITESPACE = _{ "\t" }
+ini = { SOI ~ bom? ~ (blank | pair | comment)* ~ section* }
+
+// lines
+blank = _{ "" ~ NEWLINE }
```

Comment:
Had to add `""` here so I could insert a `~` for whitespace preceding a newline.